app/configs/locationsListViewColumns.tsx

```diff
@@ -10,7 +10,7 @@ export const locationsListViewColumns: ColDataCardDef<LocationsListRow>[] =
       label: "Location ID",
       sortable: true,
       valueRenderer: (record: LocationsListRow) => (
-        <Link href={`/locations/${record.location_id}`}>
+        <Link href={`/locations/${record.location_id}`} prefetch={false}>
```

Comment:
does prefetching break if the token is expired?

Reply:
ah—thanks for catching this. i meant to stash this in a different PR. i noticed this prefetching behavior on deploy previews (prefetching doesn't run locally), and my first reaction was to disable it because our pages don't have anything to prefetch 🤷

Reply:
i'm going to rebase this commit out of this PR

Reply:
I got scared of messing up the commit history with so many PRs open. I'll make a todo to revisit prefetching.